Dockerfile

```diff
@@ -8,9 +8,8 @@ WORKDIR /bin
 
 RUN apk add --no-cache bash
 
-ADD ./install.sh /bin/install_vault_provider.sh
 COPY ./_output/secrets-store-csi-driver-provider-vault_${OS}_${ARCH}_${VERSION} /bin/secrets-store-csi-driver-provider-vault
 RUN chmod a+x /bin/secrets-store-csi-driver-provider-vault
 
```

Comment:
Not a blocker but something we need to investigate: it appears this image is running as the root user.

Reply:
Good call, this is fixable, but not trivial enough to tack onto this already multi-purpose PR. I had a quick go and it should be an easy fix but non-root users need to be given permission to the mounted volumes.